projects/eval-pqmon/src/common/afe_config.h

```diff
@@ -45,6 +45,7 @@
 #include "status.h"
 #include <stdint.h>
 #include <string.h>
+#include "nhd_c12832a1z.h"
```

Comment:
you're adding this include but i don't see anything from it being used in this commit (you do use it in the 2nd commit, so i guess it should go there)